Sources/Sentry/SentryReplayApi.m

```diff
@@ -22,18 +23,46 @@ - (void)unmaskView:(UIView *)view
 
 - (void)pause
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration pause];
 }
 
 - (void)resume
 {
-    SentrySessionReplayIntegration *replayIntegration =
-        [SentrySDK.currentHub getInstalledIntegration:SentrySessionReplayIntegration.class];
+    SentrySessionReplayIntegration *replayIntegration
+        = (SentrySessionReplayIntegration *)[SentrySDK.currentHub
+            getInstalledIntegration:SentrySessionReplayIntegration.class];
     [replayIntegration resume];
 }
 
+- (void)start
```

Comment:
`m`: Adding some tests for this here would be nice, as Codecov is complaining or is it broken?

Reply:
We have 3 tests for this function alone:
```
func testStartWithNoSessionReplay() 
func testStartWithSessionReplayRunning() 
func testStartWithBufferSessionReplay() 
```

Reply:
Yeah I thought so. Go home Codecov, you're drunk.